packages/frontend/src/components/Auction/AuctionTransaction.tsx

```diff
@@ -49,8 +50,9 @@ export const AuctionTransaction = ({ action, amount, impact, view, setView }: Au
         )}
       </TransactionWrapper>
       <TransactionStepper
-        action={action.type}
-        current={view === TxFlowSteps.Confirmation ? 'Finalized' : `${heading[action.type]}`}
+        action={action}
```

Comment:
Pass `action.type` as we don't use any other field from `TransactionAction` type.